frontend/src/components/Simulation/SimulationDetails.js

```diff
@@ -225,70 +226,83 @@ const SimulationDetails = () => {
 
   const createFieldElements = () => {
     return jobs.map((ele, idx) => (
-      <div key={`job_field_${idx}`}>
```

Comment:
Key needs to be added in the topmost div

Reply:
changes done